src/components/Loader/index.tsx

```diff
@@ -0,0 +1 @@
+export * from './Loader';
```

Comment:
Add [empty line in the end of file](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)
Fix it everywhere